### PR TITLE
Change descriptions of overwritten media types.

### PR DESCRIPTION
**Pull request by rosiel**
**GitHub Issue**: https://github.com/Islandora/documentation/issues/1541 

Other Relevant Links: 

* https://github.com/Islandora/documentation/pull/1721/files/dc48110df6aa703f2173d84b90682e92c94e41f6#r556551620
 

# What does this Pull Request do?

Changes the descriptions of the media types

# What's new?

* Changes descriptions to avoid implying that these media are for "local" files.
* Clarifies that TIFFs and JP2s should be in File, not Image media.
* Claims that these media have files "stored in Fedora". (since this is Defaults so this is true, for files added through the GUI.)

# How should this be tested?

Re-install this feature per @mjordan 's directions, here: https://github.com/Islandora/documentation/issues/1541#issuecomment-662665757

Does the "Add media" page reflect the new text?

# Additional Notes:

# Interested parties

@dannylamb @manez

### CODE REVIEW

File: config/install/media.type.audio.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ dependencies:
       - islandora_core_feature
 id: audio
 label: Audio
-description: 'A locally hosted audio file.'
+description: 'An audio file, stored in Fedora.'
```
Comment (bseeger) [inline config/install/media.type.audio.yml]: I'd recommend not using the words "stored in Fedora" as some might not be using Fedora.

**Comment by bseeger** [comment]
I have one comment above, otherwise the description changes look good to me and are definitely an improvement.

File: config/install/media.type.audio.yml (hunk 1 above)
Comment (bseeger) [reply config/install/media.type.audio.yml]: Though I see your comment in the comments above about this being defaults.  Just not sure the UI need be so specific here.  Not sure...

**Comment by rosiel** [comment]
I have been trying to make Islandora more explicit and honest about "what is in islandora and what is not". Since these media were overwritten to send items into Fedora, I thought that would be useful for people to know. of course if anyone edits how it's set up, they can/should edit the description. 
But if there's no interest in seeing 'in fedora', sure I can remove it.

**Comment by rosiel** [comment]
To be honest, i would _prefer_ that duplicate media types had been made, so that there would be "Audio" and "Islandora Audio", etc. In my opinion, knowing whether something is in fedora is an important part of managing a repository. Do you think this is something worth pursuing?

**Comment by bseeger** [comment]
Good question. I'm not sure I have enough of an opinion about it but here are some thoughts I have about multiple media types. 

If I'm using Islandora 8, I guess I would assume that all things are "in islandora" - whatever that means to your particular site setup.  Maybe that equals Fedora, maybe not. 

Is there ever a case, in the same system, where you (as a person creating items) want to have the control over whether or not they are in Fedora?   I would think that would be an overall system design - the stakeholders decide if Fedora is being used and then all things are either in it, or not, depending on the decision.  

By the time you get to the content manager adding data, the decision has been made and they add items - without having the choice of where they go. They "go in the system".   If the stakeholders wanted fedora, they go in fedora. If they didn't, they go in drupal's filesystem setup (maybe it's to disk, maybe S3). 

In that sense it would be incredibly confusing to that person to have to chose between two media types. That person doesn't have to know the tech details, they just have to know that they created the object correctly.  To me, at that point, Fedora is a tech detail. 

This is, of course, assuming the content manager doesn't know the in-depth details about the system.  That will vary, site to site. 

Basically what I'm suggesting is that whether or not an object goes into fedora is probably not made on a per-object basis, it'll most likely be made on a per-install basis.  Do we have use cases that need this decision made on a per-object basis? 

I think transparency is a very good thing and don't feel strongly about the wording above (it's fine with me if it says fedora because people can change it if they need to, as you've noted). I do think that having more then one media type per type would be confusing.

**Comment by rosiel** [comment]
Sure. I think I'd been envisioning a system where "some things are in islandora" and others weren't. That is, you could add files like hero images, marketing material, logos, etc that would help to decorate your site as it was displayed, but they would not be "in islandora". 

I think your idea of a coherent system makes more sense for users.

Good point about "wanting to know if they created the object correctly".

**Review by bseeger — APPROVED** [review]
I can see your point, about things that fill out the site.   These changes look good to me and function as expected.  I'm wondering if others want to weigh in on this or if we can just merge? 

Also, I'd been thinking about the Tiff/JP2000 issue for a while - of how to indicate to a user that you have to use File for them - so I appreciate this change.

File: config/install/media.type.audio.yml (hunk 1 above)
Comment (rosiel) [reply config/install/media.type.audio.yml]: done!

**Comment by manez** [comment]
@bseeger If you're happy with these additions, there's no need to wait for others to weigh in 👍
